instance/instance.go

```diff
@@ -30,11 +32,13 @@ var (
 // like the domain, the locale or the access to the databases and files storage
 // It is a couchdb.Doc to be persisted in couchdb.
 type Instance struct {
-	DocID      string `json:"_id,omitempty"`  // couchdb _id
-	DocRev     string `json:"_rev,omitempty"` // couchdb _rev
-	Domain     string `json:"domain"`         // The main DNS domain, like example.cozycloud.cc
-	StorageURL string `json:"storage"`        // Where the binaries are persisted
-	storage    afero.Fs
+	DocID         string `json:"_id,omitempty"`  // couchdb _id
+	DocRev        string `json:"_rev,omitempty"` // couchdb _rev
+	Domain        string `json:"domain"`         // The main DNS domain, like example.cozycloud.cc
+	StorageURL    string `json:"storage"`        // Where the binaries are persisted
+	RegisterToken string `json:"registerToken,omitempty"`
+	PasswordHash  string `json:"passwordHash,omitempty"`
```

Comment:
I think the right type for `RegisterToken` and `PasswordHash` is `[]byte` and not `string` (the bytes sequence doesn't have to respect the UTF-8 encoding)